mentoring/tests/integration/test_mentoring.py

```diff
@@ -7,3 +10,45 @@ def test_display_submit_false_does_not_display_submit(self):
         mentoring = self.go_to_page('No Display Submit')
         with self.assertRaises(NoSuchElementException):
             mentoring.find_element_by_css_selector('.submit input.input-main')
+
+
+def _get_mentoring_theme_settings(theme):
+    return {
+        'package': 'mentoring',
+        'locations': ['public/themes/{}.css'.format(theme)]
+    }
+
+
+@ddt.ddt
+class MentoringThemeTest(MentoringAssessmentBaseTest):
+    def rgb_to_hex(self, rgb):
+        r, g, b = map(int, re.search(r'rgba?\((\d+),\s*(\d+),\s*(\d+)', rgb).groups())
+        return '#%02x%02x%02x' % (r, g, b)
+
+    def assert_checkmark_color(self, color):
+        mentoring, controls = self.go_to_assessment('Theme 1')
+        question = self.expect_question_visible(0, mentoring)
+        choice_name = "Maybe not"
+
+        choices = GetChoices(question)
+        expected_state = {"Yes": False, "Maybe not": False, "I don't understand": False}
+        self.assertEquals(choices.state, expected_state)
+
+        choices.select(choice_name)
+        expected_state[choice_name] = True
+        self.assertEquals(choices.state, expected_state)
+
+        controls.submit.click()
```

Comment:
@e-kolpakov The reason this test is flaky is that an AJAX call happens here. You need to wait for it to complete.

Add this line here after `controls.submit.click()` and the tests should pass:

```python
self.wait_until_disabled(controls.submit)
```